src/molecule_plugins/azure/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -15,90 +15,98 @@
     keypair_path: "{{ lookup('env', 'MOLECULE_EPHEMERAL_DIRECTORY') }}/ssh_key"
   tasks:
     - name: Create resource group
-      azure_rm_resourcegroup:
+      azure.azcollection.azure_rm_resourcegroup:
         name: "{{ resource_group_name }}"
         location: "{{ location }}"
 
     - name: Create virtual network
-      azure_rm_virtualnetwork:
+      azure.azcollection.azure_rm_virtualnetwork:
         resource_group: "{{ resource_group_name }}"
         name: "{{ virtual_network_name }}"
         address_prefixes: "10.10.0.0/16"
 
     - name: Create subnet
-      azure_rm_subnet:
+      azure.azcollection.azure_rm_subnet:
         resource_group: "{{ resource_group_name }}"
         name: "{{ subnet_name }}"
         address_prefix_cidr: 10.10.1.0/24
         virtual_network_name: "{{ virtual_network_name }}"
 
     - name: Create key pair
-      user:
+      ansible.builtin.user:
         name: "{{ lookup('env', 'USER') }}"
         generate_ssh_key: true
         ssh_key_file: "{{ keypair_path }}"
       register: key_pair
 
     - name: Create molecule instance(s)
-      azure_rm_virtualmachine:
+      azure.azcollection.azure_rm_virtualmachine:
         resource_group: "{{ resource_group_name }}"
         name: "{{ item.name }}"
-        vm_size: Standard_A0
+        vm_size: Standard_D2_v4
         admin_username: "{{ ssh_user }}"
         public_ip_allocation_method: Dynamic
         ssh_password_enabled: false
+        managed_disk_type: Standard_LRS
```

Comment:
same here. Doesn't look like syntax fix

Reply:
@wadhah101 

I agree with @apatard 

It's better to address the non-syntax fix in another PR.

Reply:
@zhan9san 
I will open another PR for infrastructure changes. I have Reverted this change